package.json

```diff
@@ -5,10 +5,12 @@
   "module": "src/index.tsx",
   "license": "MIT",
   "scripts": {
+    "predev": "node -e 'console.clear()' && yarn install && yarn dev:clean",
     "dev": "nodemon --config config/nodemon/dev.json",
-    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development babel-node node_modules/.bin/webpack-dev-server --config config/webpack/webpack.client.babel.ts",
+    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development webpack-dev-server --config config/webpack/webpack.client.babel.ts",
     "dev:clean": "rimraf '../.build/development/{*,.*}' 'node_modules/.cache'",
-    "prod": "cross-env NODE_ENV=production BABEL_ENV=production babel-node node_modules/.bin/webpack --config config/webpack/webpack.client.babel.ts",
+    "preprod": "node -e 'console.clear()' && yarn install && yarn prod:clean",
+    "prod": "cross-env NODE_ENV=production BABEL_ENV=production webpack --config config/webpack/webpack.client.babel.ts",
```

Comment:
Using pre- hooks only makes things even more confusing in my already super-confusing setup. =)
Could we avoid that? As Microsoft folks explained in the issue thread, it is sufficient to just add spaces around semicolons `;`.
In fact I was also working on it in parallel and it indeed works.